docs/src/CommutativeAlgebra/affine_algebras.md

```diff
@@ -144,6 +144,26 @@ isfinite(hom(D, C, V))
 compose(F::AlgHom{T}, G::AlgHom{T}) where T
 ```
 
+## Subalgebras
+
+### Subalgebra Membership
+
+```@docs
+subalgebra_membership(f::S, v::Vector{S}) where S <: Union{MPolyElem{U}, MPolyQuoElem{U}} where U
```

Comment:
Meta remark: while I understand that this function is useful for implementations, as an end user, I'd rather expect to first create a subalgebra, then test containment there. I.e., instead of `subalgebra_membership(f, v)` I'd rather write  `f in sub(R, v)` or something like that. Which doesn't work here, though, as apparently we don't support `sub` for `FmpqMPolyRing` :-(

Reply:
Indeed we do not have subalgebras. I am happy to discuss this in some future. For now, please revise, push to this PR, and merge.